src/checkers/inference/solver/backend/lingeling/LingelingSolver.java

```diff
@@ -55,11 +55,11 @@ public LingelingSolver(SolverEnvironment solverEnvironment, Collection<Slot> slo
         Map<Integer, AnnotationMirror> result = new HashMap<>();
 
         this.serializationStart = System.currentTimeMillis();
-        this.convertAll();
+        this.encodeAllConstraints();
         this.serializationEnd = System.currentTimeMillis();
 
         for (Integer varSlotId : this.varSlotIds) {
-            formatTranslator.generateOneHotClauses(hardClauses, varSlotId);
+            hardClauses.addAll(formatTranslator.generateWellFormednessClauses(varSlotId));
```

Comment:
Instead of mutating the list passed as argument, return the result, and let adding logic sit inside this class.